src/Autofac.Extras.MvvmCross/AutofacPropertyInjectorOptions.cs

```diff
@@ -1,8 +1,12 @@
-﻿using Autofac.Core;
-using MvvmCross.Platform.IoC;
+﻿// <copyright file="AutofacPropertyInjectorOptions.cs" company="PlaceholderCompany">
```

Comment:
Please use the regular copyright header.

Reply:
I was looking at the develop branch to try and figure out what the regular copyright header is supposed to be, but I didn't see it anywhere. Should I perhaps just remove this copyright placeholder?

Reply:
When I remove the header, StyleCop complaints about it. Since I don't get this error with other projects, I'm assuming StyleCop is enabled for this project, is that correct?

Here's what I'm seeing when I remove the copyright header:

![image](https://user-images.githubusercontent.com/938393/40889636-d807613a-676a-11e8-92bf-a0bfd90f28d8.png)